<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Review</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keepButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep / Delete - Toggle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Marked</string>
    </property>
   </widget>
   <widget class="QListWidget" name="filelistWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="folderButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="folderLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string> image</string>
    </property>
   </widget>
   <widget class="QLabel" name="filecount">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>file count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keepALLbutton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep / Delete - Toggle ALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextfolderButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevfolderButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tognextButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>540</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Togl &gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionselect_folder"/>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionselect_folder">
   <property name="text">
    <string>select folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
